src/main.rs

```diff
@@ -185,9 +185,12 @@ impl TestConfig {
 			result: None,
 		}
 	}
+	fn set_name(&mut self, name: String) {
```

Comment:
What is this function for? I'm not a fan of setter/getters that don't do anything

Reply:
that's used in line 304. I want to change the whole approach from "do it with the global test, then the local one" to only have normal test configs and initialize them with the global config, but then I need to set the new name of course :) I realized an issue with my current approach and I think the whole TOML reading routine should be replaced. it is all WIP.